.github/workflows/ci.yml

```diff
@@ -57,15 +57,14 @@ jobs:
       - uses: actions/checkout@v4
       - run: rustup target add riscv32i-unknown-none-elf
       - run: assets/scripts/smoke.sh examples/src/bin/fib3_profiling.rs
-      - run: assets/scripts/smoke.sh examples/src/bin/hello.rs
```

Comment:
I removed `hello.rs` because from now on we need to test a *bit* more sophisticated example.